developers/weaviate/starter-guides/managing-collections/collections-scaling-limits.mdx

```diff
@@ -0,0 +1,173 @@
+---
+title: Do you have too many collections?
+sidebar_label: Scaling limits with collections
+sidebar_position: 1
+image: og/docs/more-resources.jpg
+# tags: ['performance']
+---
+
+:::info
+Added in `v1.25.34`, `v1.26.18`, `1.27.15`, `1.28.9`, `1.29.2`
+:::
+
+:::caution You hit the collections count limit?
+If you see an error that the collection count limit has been reached, it means you **can't create any more collections** because the number of collections is affecting performance. This limit exists to control indexing, storage, and management complexity.<br /><br />
```

Comment:
Two comments about this:

- I think it could be shorter.
- Technically, hitting the max collections limit does not necessary mean that `the number of collections is affecting performance`. It could be, but it's just a system limit.

Reply:
I tried to keep it short but it's still on the longer side for an admonition. I think all the info that's left is potentially relevant.